processing/src/main/java/io/druid/query/metadata/SegmentMetadataQueryConfig.java

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
can we use the header format described in https://github.com/druid-io/druid/blob/75a582974b8d3305981af30e691c67dee35acb63/server/src/test/java/io/druid/segment/realtime/plumber/RealtimePlumberSchoolTest.java for example